SB2018/2018-02-07.md

```diff
@@ -0,0 +1,35 @@
+---
+to: All Teams
+subject: January Kit Update
+---
+
+Hello there!
+
+Today's the day we're releasing the first round of kit updates for this year's competition! It's been a long time in the making, and hopefully makes your development life a lot easier!
+
+# Kit updates
+This release contains many improvements to the kit, and what you can do with it. The most notable changes are:
+- Reduce the verbosity of logs
+- New coordinate systems
+- Support for [custom arduino commands](https://docs.sourcebots.co.uk/api/servo-assembly/arduino-commands/)
+
+A more detailed changelog for this release is available from [the docs](https://docs.sourcebots.co.uk/updates/jan-2018/).
+
+To update your kit, you can either use the update file or flash the SD card. We highly recommend using the update file. Instructions on [what to do with our update file](https://docs.sourcebots.co.uk/kit/pi/update-file/), [how to flash an SD card](https://docs.sourcebots.co.uk/kit/pi/sd-card/), and the related [download links](https://docs.sourcebots.co.uk/updates/jan-2018/), are available from [the docs](https://docs.sourcebots.co.uk/kit/pi/#updating-your-pi). if you have any issues updating, be sure to consult the documentation, or [contact us](techsupport@sourcebots.co.uk).
+
+## The new coordinate systems
+In this release, we've introduced 2 new coordinate systems, and depricated an existing one.
+
+As many have noted the [`cartesian`](https://docs.sourcebots.co.uk/api/vision/coordinates/#cartesian-coordinates) system was not implemented, and would raise an exception whenever accessed. This has now been fixed.
+ 
+The `polar` coordinate system has been replaced with [`spherical`](https://docs.sourcebots.co.uk/api/vision/coordinates/#spherical-coordinates). The new system is more accurate, and better documented than polar, and we recommend updating to it as soon as possible! The available attributes are identical in `polar`, and so will likely be a simple drop-in replacement. Be sure to consult the documentation on exactly now to use the new system. Attempting to use `polar` will still work as it does now, however displays a warning message.
```

Comment:
> The new system is more accurate

I'm not sure we should claim that `spherical` is more accurate than `polar`, particularly as we're not actually fixing the accuracy in this release.

We can definitely say that it's easier to understand and better documented.